docs/tutorials/google/compare_to_calibration.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<p>Can we add a description about how to interpret these results? What's the use of doing loschmidt echo when we are also doing parallel XEB for comparison with the calibration? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Removed Loschmidt echo as discussed offline.</p>